students/csdotson/lesson09/mailroom_oo.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python3
+# Lesson 9 - Object-oriented Mailroom
+
+class Donor:
+    """ Create Donor class representing a donor and all donations """
+    def __init__(self, name, donations=[]):
+        if isinstance(name, str):
+            self.name = name
+        else:
+            raise TypeError("Name should be a string value")
+        self.donations = donations
+
+
+    @property
```

Comment:
module functions should have two lines space between functions, however methods in classes only need one line space between methods.